src/main/java/me/totalfreedom/totalfreedommod/command/Command_gtfo.java

```diff
@@ -54,6 +54,9 @@ public boolean run(CommandSender sender, Player playerSender, Command cmd, Strin
         // Rollback
         plugin.rb.rollback(player.getName());
 
+        // Rollback with CoreProtect
```

Comment:
Does this account for ASyncWorldEdit. Also see line 55, as apparently this is already done there.